public/js/selfoss-ui.js

```diff
@@ -46,6 +46,25 @@ selfoss.ui = {
                 selfoss.ui.entryMark(id, new_status.unread);
             }
         });
+    },
+
+
+    refreshStreamButtons: function(entries=false,
```

Comment:
Nice, I did not know that any browsers already support this syntax. [IE does not](https://kangax.github.io/compat-table/es6/#test-default_function_parameters), though so we’d better stay at ES5 for now.